pkgs/games/prismlauncher/default.nix

```diff
@@ -34,16 +35,16 @@ in
 
 stdenv.mkDerivation rec {
   pname = "prismlauncher";
-  version = "5.2";
+  version = "6.3";
```

Comment:
The commit here is called `prismlauncher: 6.1 -> 6.3` which might be confusing, as it changed from 5.2 to 6.3. Maybe also backport the previous bump commits to make sure the history makes sense?

Alternatively, change the commit message here.